lib/rules/max-len.js

```diff
@@ -180,6 +188,59 @@ module.exports = {
         }
 
         /**
+         * Ensure that an array exists at [key] on `object`, and add `value` to it.
+         *
+         * @param {Object} object the object to mutate
+         * @param {string} key the object's key
+         * @param {*} value the value to add
+         * @returns {void}
+         * @private
+         */
+        function ensureArrayAndPush(object, key, value) {
+            if (!Array.isArray(object, key)) {
```

Comment:
Should this be `Array.isArray(object[key])`? MDN doesn't mention a two-argument version.